lib/stack_master/template_compilers/cfndsl.rb

```diff
@@ -4,10 +4,15 @@ def self.require_dependencies
       require 'cfndsl'
     end
 
-    def self.compile(template_file_path)
-      ::CfnDsl.eval_file_with_extras(template_file_path).to_json
+    def self.compile(template_file_path, compiler_options = {})
+      if compiler_options["disable_binding"]
+        ::CfnDsl.disable_binding
+      end
```

Comment:
@thebeefcake - that's what the example implies, but we'd need some extra code to insulate the `disable_binding` setting in order to make it per-stack.  For CfnDsl it is a global setting with no toggle to turn it back on.

Since this is a deprecated param for CfnDsl, I think the simplest thing is really to just always force the bindings off since we're headed that way anyway.